src/index.js

```diff
@@ -313,7 +314,8 @@ class Dropzone extends React.Component {
         ...this.state,
         isDragActive,
         isDragAccept,
-        isDragReject
+        isDragReject,
+        open: this.open
```

Comment:
Bind the open fn to the context of the dropzone.

Reply:
It is being bound in the constructor.

Reply:
Sorry, I missed that 👍